draco/analysis/transform.py

```diff
@@ -1488,7 +1488,7 @@ def process(self, data: containers.SiderealStream | containers.RingMap):
 
         self._data_ind += 1
 
-    def process_finish(self) -> containers.SiderealStream | containers.RingMap:
+    def _make_output(self) -> containers.SiderealStream | containers.RingMap:
```

Comment:
We will need to mix the `HybridVisStream`.  I think this can just be added to the type list (both here and in the `process` method of `MixData`) and it should work without further modification to the actual pipeline task.